### PR TITLE
chore: update flag descriptions for credential flags

### PR DESCRIPTION
**Pull request by bvtujo**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: site/content/docs/commands/deploy.en.md [hunk 1]
```diff
@@ -7,31 +7,44 @@ $ copilot deploy
...
       --detach                         Optional. Skip displaying CloudFormation deployment progress.
   -e, --env string                     Name of the environment.
       --force                          Optional. Force a new service deployment using the existing image.
+                                       Not available with the "Static Site" service type.
   -h, --help                           help for deploy
+      --init-env                       Confirm initializing the target environment if it does not exist.
+      --init-wkld                      Optional. Initialize a workload before deploying it.
   -n, --name string                    Name of the service or job.
-      --no-rollback bool               Optional. Disable automatic stack
+      --no-rollback                    Optional. Disable automatic stack
```
Comment (huanjani) [inline site/content/docs/commands/deploy.en.md]: just curious: was this wrong before or did we change it?

Comment (huanjani) [reply site/content/docs/commands/deploy.en.md]: nvm I now see it was just wrong before! thx for fixing.

Comment (iamhopaul123) [reply site/content/docs/commands/deploy.en.md]: No i guess it's not published yet.

**Review by iamhopaul123 — APPROVED** [review]

File: site/content/docs/commands/deploy.en.md (hunk 1 above)
Comment (iamhopaul123) [reply site/content/docs/commands/deploy.en.md]: actually you are right. it was wrong before lol

**Comment by huanjani** [comment]
Do you think we should add something to https://aws.github.io/copilot-cli/docs/getting-started/first-app-tutorial/? 😬

**Review by huanjani — APPROVED** [review]

**Pull request merged**